src/pages/question.js

```diff
@@ -62,56 +95,83 @@ function Question() {
-                <img className="ml-2" src={Enter} alt="Space Center Logo" />
               </div>
-            </div>
+            ) : (
+              <form onSubmit={handleSubmit}>
+                <h1 className="text-white mt-36 text-5xl">{content}</h1>
+                <h1 className="text-white mt-9">
+                  {7 - curr.getDay() + " Days Left"}
+                </h1>
+                <input
+                  name="answer"
+                  className="bg-transparent border-none outline-0 placeholder:text-dove-gray max-w-xl text-gray text-2xl w-full mt-12"
+                  placeholder="Type your answer here"
+                />
+                <div className="bg-white h-1 max-w-xl min-w-min mt-2" />
+                <div className="flex gap-8 mt-10">
+                  <button
+                    className="bg-cornflower-blue rounded-lg p-4 flex text-white"
+                    type="submit"
+                  >
+                    OK
+                    <img
+                      className="pl-4 pt-1"
+                      src={Vector}
+                      alt="Space Center Logo"
+                    />
+                  </button>
+                  <div className="flex">
+                    <div className="mt-2 text-2xl text-dove-gray">press </div>
+                    <img className="ml-2" src={Enter} alt="Space Center Logo" />
+                  </div>
+                </div>
+              </form>
+            )}
             <Link to="/home">
-              <button className="bg-dove-gray p-2 lg:mt-40 mt-80 rounded-lg text-white text-sm">
+              <button className="bg-dove-gray p-2 lg:mt-40 xl:mt-10 rounded-lg text-white text-sm">
                 Go Back
               </button>
             </Link>
-          </form>
+          </div>
         </div>
-        <div className="bg-blackcurrant flex flex-col grow items-center lg:max-h-screen xl:ml-28 h-full font-[OpenSans]">
-          <form className="mt-12 gap-8 text-white flex flex-col overflow-scroll no-scrollbar">
+        <div className="bg-blackcurrant flex flex-col grow items-center lg:max-h-screen w-full h-full">
+          <h1 className="text-white text-5xl mt-8">Answers of this week</h1>
+          <form className="mt-8 gap-8 text-white flex flex-col overflow-scroll no-scrollbar">
             {answers.length === 0 ? (
               <h1> There's no answers</h1>
             ) : (
               answers.map((item) => (
                 <div
-                  className="bg-dove-gray p-4 flex flex-col rounded-3xl h-auto w-72 items-center"
+                  className="bg-dove-gray p-4 flex flex-col rounded-3xl h-auto w-72"
                   key={item.body}
                 >
-                  <div className="text-xl">{item.email}</div>
+                  <div className="text-xl text-center">{item.email}</div>
+                  <div className="text-xs text-center">
+                    {monthNames[item.time.getMonth()] +
```

Comment:
This works, but for formatting dates I recommend checking out the [`toLocaleDateString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) method